File: stress/t/int_fib.c
#include <stdio.h>
#include <stdint.h>
int main(void){
  int64_t a=0,b=1,t,i=0,n=90;
  while(i<n){ t=(int64_t)((uint64_t)a+(uint64_t)b); a=b; b=t; i=i+1; }
  printf("fib90="); printf("%ld",(long)a); printf("\n");
  printf("fib91="); printf("%ld",(long)b); printf("\n");
  return 0;
}

File: stress/t/int_fib.f
      PROGRAM INTFIB
      INTEGER*8 A,B,T,I,N
      A=0
      B=1
      N=90
      I=0
   10 IF (I.GE.N) GOTO 20
      T=A+B
      A=B
      B=T
      I=I+1
      GOTO 10
   20 CONTINUE
      WRITE(*,'(A,I0)') 'fib90=',A
      WRITE(*,'(A,I0)') 'fib91=',B
      END
